wuvt/auth/auth_manager.py

```diff
@@ -0,0 +1,119 @@
+from flask import abort, make_response, session, url_for
+from flask_login import current_user, LoginManager
+from flask_oidc import OpenIDConnect
+from functools import wraps
+
+from wuvt.models import User
+from .blueprint import bp
+
+
+class AuthManager(object):
+    def __init__(self, app=None):
+        self.bp = bp
+        self.all_roles = [
+            'admin',
+            'content',
+            'business',
+            'library',
+            'missioncontrol',
+        ]
+
+        if app is not None:
+            self.init_app(app)
+
+    def init_app(self, app):
+        from . import views
+
+        self.app = app
+
+        app.config.setdefault('AUTH_METHOD', 'local')
+        app.config.setdefault('AUTH_ROLE_GROUPS', {
+            'admin': ['webmasters'],
```

Comment:
Yeah, the default is supposed to be different. ACLs are somewhat security-sensitive so I think it's best to avoid publishing them in their entirety if we can.